lang/attribute/event/src/lib.rs

```diff
@@ -81,6 +81,35 @@ pub fn emit(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
     })
 }
 
+#[proc_macro]
+pub fn emit_cpi(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
```

Comment:
Can we add documentation for this macro?

Reply:
Will do once I write this up as an sRFC on forum.solana.com. There's still an additional argument that's necessary for the invoke, which is the Instructions sysvar account.

Once this is ironed out, then yes I will add documentation here

Reply:
Wait if you need SysvarInstructions, then you should just use a log authority PDA

Reply:
Yeah I agree. Just had to check